clients/js/src/index.ts

```diff
@@ -66,8 +66,9 @@ let CohereAiApi: any;
 
 export class CohereEmbeddingFunction {
   private api_key: string;
+  private model: string;
 
-  constructor(cohere_api_key: string) {
+  constructor(cohere_api_key: string, cohere_model?: string) {
```

Comment:
can we renamed this to `model`? `cohere_model` is repetitive, and diverges from the underlying SDK from Cohere

Reply:
Yes, sounds good. It is simpler. Should we rename the OpenAIEmbeddingFunction() parameters as well. 

````
export class OpenAIEmbeddingFunction {
  private api_key: string;
  private org_id: string;
  private model: string;
  constructor(openai_api_key: string, openai_model?: string, openai_organization_id?: string) {
  ...
  ````

Reply:
It should probably be standard... but I also don't want to cause a breaking change over it right now. So let's leave as-is